doc/glossary.rst

```diff
@@ -675,6 +675,19 @@ General Concepts
         but not maintained by the core developer team.
         See https://scikit-learn-contrib.github.io.
 
+    scikit-learn enhancement proposals
+    SLEP
+    SLEPs
+        Changes to the API principles and changes to dependencies or supported
+        versions happen via a :ref:`SLEP <slep>`_ and follows the
```

Comment:
I believe you should not use the trailing `_` when using an internal `:ref:` link. The trailing `_` shall only be used for external URL links.